test/MUXControlsTestApp/TestInventory.cs

```diff
@@ -34,58 +34,165 @@ static TestInventory()
         {
             Tests = new List<TestDeclaration>()
             {
-#if !BUILD_LEAN_MUX_FOR_THE_STORE_APP
+// #if FEATURE_SCROLLER_ENABLED && FEATURE_COLOR_PICKER_ENABLED && FEATURE_NAVIGATION_VIEW_ENABLED && FEATURE_RATING_CONTROL_ENABLED
+//              {"Leak", typeof(LeakTestPage)},
+//#endif
+
+#if FEATURE_COLOR_PICKER_ENABLED
```

Comment:
I think i'm going to change these names to match the ones in the native (like COLORPICKER_INCLUDED) or change the other ones to match this. Let me know if you prefer one over the other.

Reply:
removed most of then, I think we can avoid these now that we are using reflection.

---
In reply to: [302677541](https://github.com/microsoft/microsoft-ui-xaml/pull/956#discussion_r302677541) [](ancestors = 302677541)